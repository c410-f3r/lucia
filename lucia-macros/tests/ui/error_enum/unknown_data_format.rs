#[lucia_macros::pkg(api(Foo), data_format(fdsdfs))]
mod pkg {
  #[pkg::req_data]
  struct Req;

  #[pkg::res_data]
  struct Res;
}

fn main() {
}
