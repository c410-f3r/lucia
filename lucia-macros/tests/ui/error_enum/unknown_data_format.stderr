error: Unknown data format.
 --> tests/ui/error_enum/unknown_data_format.rs:4:1
  |
4 | #[lucia_macros::pkg(api(Foo), data_format(fdsdfs))]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `lucia_macros::pkg` (in Nightly builds, run with -Z macro-backtrace for more info)
